shapely/constructive.py

```diff
@@ -926,6 +927,61 @@ def simplify(geometry, tolerance, preserve_topology=True, **kwargs):
         return lib.simplify(geometry, tolerance, **kwargs)
 
 
+@multithreading_enabled
+def polygonhull_simplify(
```

Comment:
I suppose that given the aim to match the PostGIS naming, this should be called `simplify_polygon_hull` after [`ST_SimplifyPolygonHull`](https://postgis.net/docs/ST_SimplifyPolygonHull.html).